packages/design-system-react/src/utils/tw-merge.ts

```diff
@@ -42,20 +42,16 @@ export const twMerge = extendTailwindMerge({
         },
       ],
       'font-weight': [
-        // Standard Tailwind font weights
         'font-thin',
         'font-extralight',
         'font-light',
         'font-normal',
+        'font-regular',
         'font-medium',
         'font-semibold',
         'font-bold',
         'font-extrabold',
         'font-black',
-        // Custom typography classes
-        {
-          font: variantClassGroups,
-        },
```

Comment:
We have removed variant based font weight classnames. e.g. `font-s-display-lg` so these are no longer needed